src/pages/workspace/expensifyCard/issueNew/IssueNewCardPage.tsx

```diff
@@ -30,6 +30,12 @@ function IssueNewCardPage({policy, route}: IssueNewCardPageProps) {
         Card.startIssueNewCardFlow(policyID);
     }, [policyID]);
 
+    useEffect(() => {
+        return () => {
+            Card.clearIssueNewCardFlow();
```

Comment:
Why do we need to clear data here? We already cleared data if the API failed

Reply:
I noticed that when we close RHP during the flow and go to another workspace and start the flow for issuing card we have the data from the previous flow. This may lead to an error when i.e. we have chosen a assignee that is not a member of the new workspace. To avoid such errors I think it's better to clear the flow everytime it's closed. 

cc @mountiny